docs/src/domains/steps/tls.md

```diff
@@ -75,11 +78,18 @@ For the new certificate to be taken into account, you need to [redeploy the envi
 platform environment:redeploy
 ```
 
-{{< note theme="info" title="Success!" >}}
+Your site is now live and accessible to the world as soon as the DNS propagates.
 
-Your site should now be live and accessible to the world (as soon as the DNS propagates).
+If something isn't working see the [troubleshooting guide](/domains/troubleshoot.md) for common issues.
+If that doesn't help, [contact support](../../overview/get-support.md).
 
-{{< /note >}}
+### How to change the private key format
 
-If something isn't working see the [troubleshooting guide](/domains/troubleshoot.md) for common issues.
-If that doesn't help, feel free to [contact support](../../overview/get-support.md).
+Your certificate's private key has to be in PKCS#1 style, and start with `BEGIN RSA PRIVATE KEY`.
+If it starts with `BEGIN PRIVATE KEY`, it's PKCS#8 and bundled with the identifier for key type.
+
+To convert the RSA key from PKCS#8 to PKCS#1, run:
+
+```bash
+openTLS rsa -in private.key -out private.rsa.key
```

Comment:
:question: Not with variables? I'd guess these are the defaults, but are they necessarily followed? Is it necessary to rename the key?